core/src/components/searchbar/searchbar.tsx

```diff
@@ -54,7 +54,7 @@ export class Searchbar implements ComponentInterface {
   /**
    * Set the cancel button icon. Only applies to `md` mode.
    */
-  @Prop() cancelButtonIcon = 'md-arrow-back';
+  @Prop() cancelButtonIcon = `md-${config.get('backButtonIcon', 'arrow-back')}`;
```

Comment:
```suggestion
  @Prop() cancelButtonIcon = config.get('backButtonIcon', 'md-arrow-back');
```

Reply:
Ahhh, I see. `md-` in `md-arrow-back` is redundant since ion-icon is setting the mode to md. This got me confused.